src/Lykke.Blockchains.Integrations.Contract/Common/Base58String.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Text;
+using System.Text.RegularExpressions;
+using JetBrains.Annotations;
+using Newtonsoft.Json;
+using SimpleBase;
+
+namespace Lykke.Blockchains.Integrations.Contract.Common
+{
+    /// <summary>
+    /// Some binary object or probably json object encoded as base58 string.
+    /// </summary>
+    [PublicAPI]
+    [JsonConverter(typeof(Base58StringJsonConverter))]
+    public sealed class Base58String
```

Comment:
Looks like, it also can be implicitly converted to string (and explicitly from string)

Reply:
Since this class is inteded to convert strings to base58, it's not clear if implicit type conversion converts a plain string or just gets base58 string. I had have such implementation before PR, it was confusing.